src/app/containers/Include/index.jsx

```diff
@@ -30,7 +31,20 @@ const IncludeContainer = props => {
   const { enabled } = useToggle('include');
 
   if (!enabled) return null;
-  const { href, type } = props;
+  const { showEmbedError, href, type } = props;
+
+  if (showEmbedError) {
+    return (
+      <EmbedError
+        message="Sorry, we can’t display this part of the story on this lightweight mobile page."
+        link={{
+          text: 'View the full version of the page to see all the content.',
+          href: 'https://www.bbc.co.uk/',
+        }}
+        fillViewport
+      />
```

Comment:
Just a couple of things with regards to EmbedError:
- We shouldn't be using `fillViewport` in this context.
- We should be wrapping this component in a grid item to ensure it has the correct margins - it's fine to do this in the return statement.
- `href`'s value should be a link to the Canonical page, plus the Include ID.